spec/lang/representation.md

```diff
@@ -371,8 +385,11 @@ Similarly, on `Pointer` we say that adding provenance makes it more defined:
 ```rust
 impl<Provenance> DefinedRelation for Pointer<Provenance> {
     fn le_defined(self, other: Self) -> bool {
-        self.addr == other.addr &&
-            match (self.provenance, other.provenance) {
+        self.thin_pointer.addr == other.thin_pointer.addr &&
+        // Feedback wanted: Using `Option<PointerMeta> as DefinedRelation` would be okay aswell, 
+        // since all antecedents garantee that either both or none have metadata in the first place?
+        self.metadata == other.metadata && 
```

Comment:
In the future PointerMeta will contain vtable pointers that can have provenance, so it's probably better to define a `DefinedRelation` on `PointerMeta`.